arcgis-ios-sdk-samples/Edit data/Edit and sync features/EditAndSyncFeaturesViewController.swift

```diff
@@ -141,6 +142,9 @@ class EditAndSyncFeaturesViewController: UIViewController {
             self.barButtonItem.isEnabled = false
             self.instructionsLabel.text = "Tap on a feature"
             self.mapView.touchDelegate = self
+            self.mapView.interactionOptions.isPanEnabled = false
+            self.mapView.interactionOptions.isZoomEnabled = false
+            self.mapView.interactionOptions.isRotateEnabled = false
```

Comment:
Rather than disabling each of these interactions individually, could user interaction be disabled on the map view itself?

Reply:
No, user interaction must be enabled so that the user can select and move features.

Reply:
Ah, that makes sense. Thanks!